### ISSUE TITLE
Datepicker has no day for selection with default start and last date

### ISSUE THREAD

**Issue by OleksandrFURHALIUK** [flet-dev/flet#4639]
### Duplicate Check

- [x] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

When create DatePicker with default parameters and add to page there are no day for selection.
![image](https://github.com/user-attachments/assets/d0fbfb73-9875-437c-9bb1-75aedd23baa3)
![image](https://github.com/user-attachments/assets/c7e3f98a-a88e-4b5b-b608-1f0bd993a57a)

here is type hint PyCharm ide and I think that default values for last date and first date must be interchanged.
![image](https://github.com/user-attachments/assets/aa557bce-000e-46b5-817c-9f81658903a2)

When  explicitly specify the first and last date everything works fine as on next screen
![image](https://github.com/user-attachments/assets/0476a9a6-0eab-470a-b910-3c2b91a475d7)

![image](https://github.com/user-attachments/assets/0d181934-2dcd-4bf9-a726-064c81fd2cf9)

Is this a small bug or is it supposed to work like this?

### Code sample

<details open><summary>Code</summary>

```python
from datetime import datetime

import flet as ft

def main(page: ft.Page) -> None:

    date_picker = ft.DatePicker()
    page.add(date_picker)

    page.open(date_picker)
    page.update()

ft.app(target=main)
```

</details>

### To reproduce

run the code above

### Screenshots / Videos

<details open>
<summary>Captures</summary>

[Upload media here]

</details>

### Operating System

Linux

### Operating system details

Ubuntu 22.04

### Flet version

0.25.2

### Regression

I'm not sure / I don't know

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by excessivechaos** [comment]
Yeah I have this same issue.  Seems to be a regression as the defaults worked fine in prior version.  Explicitly specify start and end dates so end is > than start is the workaround.
